css/styles.css

```diff
@@ -695,3 +697,101 @@ li {
     border-radius: 1rem;
   }
 }
+
+/* Pop Up */
+.pop-up {
+  margin: 2rem 0;
+  padding-bottom: 1rem;
+  background-color: var(--white);
+  position: fixed;
+  z-index: 1;
+  top: 0;
+}
+
+.pu-title {
+  justify-content: space-between;
+  margin-bottom: 5rem;
+  margin-top: 2rem;
+}
+
+.bt-popup {
+  border: none;
+  background-color: var(--white);
+  padding-top: 1rem;
+}
+
+.bt-popup:hover {
+  cursor: pointer;
+}
+
+.pu-buttons {
+  justify-content: space-between;
+}
+
+.pu-img1 {
+  width: 30%;
+  padding-left: 1rem;
+}
+
+.pu-img2 {
+  width: 21%;
+  padding-left: 1rem;
+}
+
+.pu-list {
+  margin: 2rem 0;
+}
+
+.pu-ul li {
+  font-family: var(--poppins);
+  color: var(--maastrichtBlue);
+  font-size: 1.3rem;
+  font-weight: 600;
+  margin-right: 1rem;
+  padding: 1rem 1.2rem;
+  border-style: solid;
+  border-color: var(--coolGrey);
+  border-width: 0.1rem;
+}
+
+.pu-title h2 {
+  font-family: var(--roboto);
+  font-weight: 700;
+  color: var(--maastrichtBlue);
+  font-size: 3.2rem;
+}
+
+@media (min-width: 768px) {
+  .pop-up {
+    left: 7%;
+    right: 0;
+    top: 10%;
+    padding-bottom: 2rem;
+  }
+
+  .pu-main {
+    display: flex;
+  }
+
+  .pu-desc {
+    width: 50%;
+    margin-left: 2rem;
+  }
```

Comment:
- [ ] Nice work implementing the details popup windows :clap: However It would be nice to bring the buttons down on the same line as the image to follow the Figma design.
### Your design :point_down: 
![Screenshot from 2023-05-17 17-12-03](https://github.com/lRebornsl/portfolio/assets/91142432/9cb21697-84ec-452e-b91d-0441933bb4d7)
### What it is supposed to look like :point_down: 
![Screenshot from 2023-05-17 17-33-36](https://github.com/lRebornsl/portfolio/assets/91142432/006d1d8b-d61b-43b5-b6cc-682179bcd236)